modin/pandas/groupby.py

```diff
@@ -538,7 +544,7 @@ def try_get_str_func(fn):
                 )
 
             if any(i not in self._df.columns for i in func_dict.keys()):
-                from pandas.core.base import SpecificationError
+                from modin._compat.core.pd_common import SpecificationError
```

Comment:
Why do we need to expose the exceptions in `modin._compat`?

Reply:
because they're in different place in `pandas` - they're in `pandas.core.base` in pre-1.5 and in `pandas.errors` in 1.5

Reply:
I see, thanks!